extend-expect.d.ts

```diff
@@ -1,16 +1,17 @@
 import type { ReactTestInstance } from 'react-test-renderer';
+import type { ImageStyle, TextStyle, ViewStyle } from 'react-native';
 
 declare global {
   namespace jest {
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     interface Matchers<R, T> {
       toBeDisabled(): R;
-      toContainElement(element: ReactTestInstance | null): R;
+      toContainElement(element: ReactTestInstance | null | undefined): R;
       toBeEmpty(): R;
-      toHaveProp(attr: string, value?: any): R;
+      toHaveProp(attr: string, value?: unknown): R;
       toHaveTextContent(text: string | RegExp, options?: { normalizeWhitespace: boolean }): R;
       toBeEnabled(): R;
-      toHaveStyle(style: object[] | object): R;
+      toHaveStyle(style: StyleProp<ViewStyle> | StyleProp<TextStyle> | StyleProp<ImageStyle>): R;
```

Comment:
This probably should be more inclusive for passing styles.
```suggestion
      toHaveStyle(style: StyleProp<ViewStyle | TextStyle | ImageStyle>): R;
```